.github/workflows/python-package.yml

```diff
@@ -121,9 +121,10 @@ jobs:
         export DACE_testing_single_cache=1
         export COVERAGE_RCFILE=`pwd`/.coveragerc
         export PYTHON_BINARY="coverage run --source=dace --parallel-mode"
+        export LIBRARY_PATH=$LIBRARY_PATH:/usr/lib/x86_64_linux-gnu
```

Comment:
It could be done like that. But every time hardware emulation is run, this is needed. So maybe it should be moved to `xilinx_test.sh`, so it's emphasized that the tests following that export is hardware emulation?

Reply:
As discussed, change to pass the environment variable to the specific call to `v++` that needs this

Reply:
Moved to `tests/xilinx_test.sh` in https://github.com/carljohnsen/dace/commit/b475c67e0b04c6eeb41b1ebf197910e7f14f64f3